yarn.lock

```diff
@@ -7612,6 +7640,19 @@ yargs@16.2.0, yargs@^16.1.1, yargs@^16.2.0:
     y18n "^5.0.5"
     yargs-parser "^20.2.2"
 
+yargs@17.5.1:
```

Comment:
unfortunate that we are now adding another yargs + all this other CLI color crap to our deps with this PR.

At the very least, we could see about upgrading to latest yargs to reduce this a bit.